app/src/main/java/com/practicum/playlistmaker/AppThemeMode.kt

```diff
@@ -0,0 +1,40 @@
+package com.practicum.playlistmaker
+
+import android.app.Application
+import android.content.SharedPreferences
+import android.content.res.Configuration
+import androidx.appcompat.app.AppCompatDelegate
+
+class AppThemeMode : Application() {
+
+    var darkTheme = false
+    lateinit var sharedPreferences: SharedPreferences
+
+    override fun onCreate() {
+        super.onCreate()
+
+        sharedPreferences = getSharedPreferences("SHARED_PREFERENCE", MODE_PRIVATE)
+        val darkModeOn = resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK == Configuration.UI_MODE_NIGHT_YES
+        darkTheme = sharedPreferences.getBoolean("DARK_THEME_KEY", darkModeOn)
```

Comment:
```suggestion
        darkTheme = sharedPreferences.getBoolean(DARK_THEME_KEY, darkModeOn)
```
В данной строчке используется строка, вместо нужного ключа для чтения значения темы (в 32 строке используется именно этот ключ для записи значения темы)

Reply:
Да, понял, невнимательность. Спасибо!